### PR TITLE
USHIFT-315: Add swap space to the default development setup

### PR DESCRIPTION
**Pull request by dhellmann**
The system should perform better with some explicit swap space.

/assign @ggiguash

### CODE REVIEW

File: scripts/devenv-builder/create-vm.sh [hunk 1]
```diff
@@ -31,17 +31,33 @@ DATAVOLSIZE=$7
 [[ ! "${DISKSIZE}" =~ ^[0-9]+$ ]]    && usage "Invalid disk size: '${DISKSIZE}'"
 [[ ! "${DATAVOLSIZE}" =~ ^[0-9]+$ ]] && usage "Invalid data volume size: '${DATAVOLSIZE}'"
 
+# It would be ideal to let the system choose the recommended amount of
+# swap automatically but then we could not compute the SYSROOTSIZE
+# below so we have to apply the logic ourselves based on
+# https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/managing_storage_devices/getting-started-with-swap_managing-storage-devices
+if [ $RAMSIZE -le 2 ]; then
+    SWAP=$(( $RAMSIZE * 2 ))
```
Comment (ggiguash) [inline scripts/devenv-builder/create-vm.sh:39]: Nit: Could we use SWAPSIZE name to be consistent with the rest of the style?

**Comment by ggiguash** [comment]
/retitle USHIFT-315: Add swap space to the default development setup

**Comment by ggiguash** [comment]
@dhellmann, can you update the devenv_rhel8_auto.md doc to explain how swap takes free space away from the total disk size?

**Comment by dhellmann** [comment]
Let's take #894 instead

/close

**Pull request closed without merging**